base/src/entity/channel/message.rs

```diff
@@ -92,6 +93,37 @@ impl From<Message> for MessageEntity {
     }
 }
 
+impl From<(MessageUpdate, MessageEntity)> for MessageEntity {
```

Comment:
The order in the tuple should be reversed, as it's a message entity being updated. But, this `From` is still weird.. maybe a method `update` that takes `&mut self`?